terraform/modules/secret/variables.tf

```diff
@@ -1,5 +1,5 @@
-variable "name" {
-  description = "The name of the secret in Secrets Manager"
+variable "name_prefix" {
+  description = "The name prefix of the secret in Secrets Manager"
```

Comment:
Could you add a default for this? Since it's really only for testing and not something that normal users should care about we should try to keep it under the hood as much as possible.